packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -99,6 +99,13 @@ export const CurrentProject = objectType({
       },
     })
 
+    t.boolean('forceReconfigureProject', {
```

Comment:
Rather than surfacing an additional property, would it be better to modify `isCtConfigured` and `isE2eConfigured` to take into accound the `forceReconfigureProject` flag e.g.
 `isCtConfigured: { resolve: ctx.lifecycleManager.isTestingTypeConfigured('component') && !ctx.coreData.forceReconfigureProject }`.
 
 Also, do you know if the the project setup should be testingType specific? The user could have selected component testing, hit the reconfigureProject and then hit the back button in the launchpad project setup page and click on e2e testing which would bring them into e2e setup.